zetaclient/chains/evm/signer/outbound_data.go

```diff
@@ -1,21 +1,19 @@
 package signer
 
 import (
-	"context"
 	"encoding/base64"
 	"encoding/hex"
-	"errors"
 	"fmt"
 	"math/big"
 
 	ethcommon "github.com/ethereum/go-ethereum/common"
+	"github.com/pkg/errors"
```

Comment:
```suggestion
	"errors"
```
Not sure why this github.com/pkg/errors continues being added across the codebase, are you using some formatter? 🤔 
We have to use errors from the stdlib, which is the maintained implementation.

Reply:
Because @swift1337 likes using `errors.Wrap()` which is not in the stdlib.`fmt.Errorf("...: %w", err)` is what became standardized afaik.